sdk/src/client/node_api/indexer/query_parameters.rs

```diff
@@ -124,12 +118,12 @@ pub struct BasicOutputsQueryParameters {
     pub timelocked_before: Option<SlotIndex>,
 }
 
-impl_query_parameters_methods!(BasicOutputsQueryParameters, BasicOutputsQueryParametersBuilder);
+impl_query_parameters_methods!(BasicOutputsQueryParameters);
 
-impl BasicOutputsQueryParametersBuilder {
+impl BasicOutputsQueryParameters {
     /// Sets `.address(address).has_expiration(false).has_expiration(false).has_storage_deposit_return(false)` to only
     /// get outputs that can be unlocked by the address without potential further restrictions.
-    pub fn only_address_unlock_condition(&mut self, address: impl Into<Bech32Address>) -> &mut Self {
+    pub fn only_address_unlock_condition(self, address: impl Into<Bech32Address>) -> Self {
         self.address(address.into())
             .has_expiration(false)
             .has_expiration(false)
```

Comment:
supposed to be `has_timelock`?

Reply:
Yes thanks, not sure how I missed this :joy: